src/PSSparseServerInterface.cpp

```diff
@@ -121,7 +121,8 @@ void PSSparseServerInterface::get_lr_sparse_model_inplace(const SparseDataset& d
   
   //4. receive weights from PS
   uint32_t to_receive_size = sizeof(FEATURE_TYPE) * num_weights;
-  //std::cout << "Model sent. Receiving: " << num_weights << " weights" << std::endl;
+  // std::cout << "Model sent. Receiving: " << num_weights << " weights" <<
```

Comment:
There are a few prints that are commented in this PR. I would either remove them or preferentially putting them inside debug ifdefs.

Reply:
Fixed